middleware/markdown/markdown.go

```diff
@@ -99,74 +70,67 @@ func (md Markdown) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error
 			continue
 		}
```

Comment:
Good point, I'll implement "search and exit if not match/found early" instead of the loop I have.